src/hotspot/share/oops/constantPool.cpp

```diff
@@ -1276,6 +1276,33 @@ void ConstantPool::unreference_symbols() {
   }
 }
 
+// Returns true if the current mismatch is due to a resolved/unresolved
+// class pair. Otherwise, returns false.
+bool ConstantPool::is_unresolved_class_mismatch(int index1,
```

Comment:
Has this been moved verbatim from jvmtiRedefineClasses.cpp?

There are a couple of style nits with that existing code that don't fit this file:
- parameters should line up ie. const under int
- no comment on the closing brace of the method body.

Reply:
> Has this been moved verbatim from jvmtiRedefineClasses.cpp?

yes

> There are a couple of style nits with that existing code that don't fit this file:
> 
> * parameters should line up ie. const under int
> * no comment on the closing brace of the method body.

Will adjust accordingly